tests/trainer/test_trainer.py

```diff
@@ -546,6 +547,40 @@ def test_trainer_min_steps_and_epochs(tmpdir):
     assert trainer.global_step >= math.floor(num_train_samples * 1.5), "Model did not train for at least min_steps"
 
 
+def test_trainer_min_steps_and_min_epochs_not_reached(tmpdir, caplog):
```

Comment:
shall we rather integrate it in an exiting test?
CC: @carmocca

Reply:
hmm, it doesn't fit well with the other test. I don't see an easy way because we need a particular set of trainer arguments.